hubblestack_nova/modules/win_reg.py

```diff
@@ -40,7 +40,6 @@ def audit(data_list, tags, verbose=False, show_profile=False, debug=False):
         log.debug('registry audit __data__:')
         log.debug(__data__)
         log.debug('registry audit __tags__:')
-        log.debug(__tags__)
```

Comment:
Can you add this line back in? It might have been deleted as part of my merge conflict resolution, but we want it.